### PR TITLE
feat(repository): support insert many 

### PR DESCRIPTION
**Pull request by simylein**

### CODE REVIEW

**Review by Slowlydev — APPROVED** [review]
Looks like a pretty random braking change, but code wise its good 👍🏻

File: lib/repository/repository.ts [hunk 1]
```diff
@@ -14,6 +15,7 @@ type Repository<T extends IdEntity> = {
 	find: <S extends keyof T>(options?: FindOptions<T, S>) => Promise<Pick<T, S>[]>;
 	findOne: <S extends keyof T>(options: T['id'] | FindOneOptions<T, S>) => Promise<Pick<T, S> | null>;
 	insert: (data: InsertData<T>) => Promise<{ id: T['id'] }>;
+	insertMany: (data: InsertData<T>[]) => Promise<{ id: T['id'] }[]>;
```
Comment (Slowlydev) [inline lib/repository/repository.ts:18]: Technically it should be multi right? Too keep it consistent?

**Review by Slowlydev — APPROVED** [review]

File: lib/repository/repository.ts (hunk 1 above)
Comment (simylein) [reply lib/repository/repository.ts:18]: No, I had to chose multi because of naming collisions, I would have loved to call the db function insertMany

**Comment by simylein** [comment]
Do you think the unit tests are representative?

**Comment by Slowlydev** [comment]
> Do you think the unit tests are representative? Do we need some more?

Is it possible to fail? If yes do a negative test. Else if it only can insert and have basic things like updatedAt, createdAt & undefined handling. That's it yeah.

**Pull request merged**